tests/test_sample.py

```diff
@@ -171,11 +173,11 @@ def test_paulix_pauliy(self, device, shots, tol):
         dev.apply("CNOT", wires=[1, 2], par=[])
 
         dev._obs_queue = [
-            qml.PauliX(wires=[0], do_queue=False) @ qml.PauliY(wires=[2], do_queue=False)
+            Tensor(["PauliX", "PauliY"], [[0], [2]], [[], []], qml.operation.Sample)
         ]
 
-        for idx in range(len(dev._obs_queue)):
-            dev._obs_queue[idx].return_type = qml.operation.Sample
```

Comment:
How come these were removed? Why not just delete?

Reply:
This is because I was making a PL core tensor observable assumption; by removing the dependence on the PL core tensor branch, these are no longer needed.